dts/bindings/interrupt-controller/sifive,clic.yaml

```diff
@@ -0,0 +1,19 @@
+# Copyright (c) 2021, Tokita, Hiroshi
+# SPDX-License-Identifier: Apache-2.0
+
+description: SiFive CLIC interrupt controller
+
+compatible: "sifive,clic"
+
+include: [interrupt-controller.yaml, base.yaml]
```

Comment:
Also is this interrupt controller schema a mistaken git add?  I don't see that it's relevant to the timer driver change?

Reply:
The DT_DRV_COMPAT sifive_clic is dependent on this file, as the filtering is (wrongly, imitating the previously existing code as it was assumed correct) based on the interrupt controller scheme.

Reply:
Ok actually this is getting more confusing, why is sifive,clint0 in timer, that's just wrong.